util/argo/argo.go

```diff
@@ -115,6 +115,43 @@ func FilterByProjects(apps []argoappv1.Application, projects []string) []argoapp
 	return items
 }
 
+func FilterByPath(apps []argoappv1.Application, path string) []argoappv1.Application {
+	filteredApps := make([]argoappv1.Application, 0)
+	for _, app := range apps {
+		if app.Spec.Source != nil && (app.Spec.Source.Path == path || strings.HasPrefix(app.Spec.Source.Path, path+"/")) {
```

Comment:
```suggestion
		if app.Spec.Source != nil && (app.Spec.Source.Path == path || strings.HasPrefix(app.Spec.Source.Path, path+"/")) {
```
why are you checking for `strings.HasPrefix()` here?

Reply:
Hi @nitishfy , apologies I'm a bit new around here.

I've put it like that so it compares the exact path along with subdirectories and all ?

Is it not required ?

Reply:
You'd need to check for absolute and relative path here instead of just checking if a path consist of `/`. Isn't this the intended behaviour? cc @agaudreault